### ISSUE TITLE
Missing import of enum const in generated mock when using split mode

### ISSUE THREAD

**Issue by severinh** [anymaniax/orval#1201]
## What are the steps to reproduce this issue?

1. Use an OpenAPI schema that contains a top-level enum type that is referenced in a response. Minimal example:

```yaml
openapi: 3.1.0
info:
  title: Missing import
paths:
  /dog:
    get:
      responses:
        200:
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Dog'
components:
  schemas:
    Dog:
      type: object
      properties:
        type:
          $ref: '#/components/schemas/DogType'
    DogType:
      type: string
      enum:
        - Labradoodle
        - Dachshund

```

2. Use an Orval config with `mode: 'split'` and `mock: true`.

## What happens?

See that the generated `missing-import.msw.ts` file has a TypeScript error `Cannot find name 'DogType'.`

```typescript
import { faker } from '@faker-js/faker';
import { HttpResponse, delay, http } from 'msw';
import type { Dog } from './missingImport.schemas';  // DogType is not imported

export const getGetDogMock = (overrideResponse: any = {}): Dog => ({
  type: faker.helpers.arrayElement([
    faker.helpers.arrayElement(Object.values(DogType)),   // DogType is not imported
    undefined,
  ]),
  ...overrideResponse,
});
```

## What were you expecting to happen?

The generated MSW file should pass the type check. One of two things should happen:

1. Either, the imports should be extended to include the DogType const, such as `import { type Dog, DogType } from './missingImport.schemas';`, or
2. The list of enum values should be inlined in the arrayElement call, such as `faker.helpers.arrayElement(['Labradoodle', 'Dachshund'])`

## Any logs, error output, etc?

N/A

## Any other comments?

Thanks a lot for creating Orval! Apart from this bug, using Orval has been a great experience. We're impressed by the depth of the customisability, and how clean the generated types are.

## What versions are you using?

**Operating System:** MacOS 14.2.1
**Package Version:** 6.24.0
**Browser Version:** N/A

**Comment by soartec-lab** [comment]
@severinh 
Thank you for made this report, this issue made by https://github.com/anymaniax/orval/pull/1171. 
Please wait as I address this issue and include the fix in the next version.

**Comment by soartec-lab** [comment]
Similar to https://github.com/anymaniax/orval/issues/1202

**Comment by soartec-lab** [comment]
@severinh 
Hi, this issue fixed by #1205 
The next version will have the import working fine, so check it out when it's released.

**Comment by severinh** [comment]
> @severinh Hi, this issue fixed by #1205 The next version will have the import working fine, so check it out when it's released.

That is great news! Thanks for fixing this issue so quickly, @soartec-lab! I'll check it out as soon as there is a release.

**Comment by soartec-lab** [comment]
Hi, @severinh 

Since we released "v6.25.0" this issue has been fixed 👍
